src/domain/enrolment/CreateEnrolmentPage.tsx

```diff
@@ -28,6 +32,21 @@ const CreateEnrolmentPage: React.FC = () => {
   };
 
   const event = eventData?.event;
+  const eventLocationId = event?.location?.id || '';
+  const selectedOccurrrences = Array.isArray(occurrences)
+    ? occurrences
+    : [occurrences];
+
+  const allOccurrences: OccurrenceFieldsFragment[] =
+    event?.pEvent?.occurrences.edges.map(
```

Comment:
Maybe this would be little bit cleaner if this edge-stuff would be in its own helper function?